File: 프로그래머스/0/120864. 숨어있는 숫자의 덧셈 （2）/숨어있는 숫자의 덧셈 （2）.java
class Solution {
    public int solution(String my_string) {
        int answer = 0;
        String[] strList = my_string.split("[^0-9]+");

        for (String str : strList) {
            if (!str.isEmpty()) {
                answer += Integer.parseInt(str);
            }
        }
        
        return answer;
    }
}

File: 프로그래머스/0/120864. 숨어있는 숫자의 덧셈 （2）/숨어있는 숫자의 덧셈 （2）.js
function solution(my_string) {
    let answer = 0;
    let strList = my_string.split(/[^0-9]+/);

    for (let str of strList) {
        if (str !== "") { 
            answer += parseInt(str);
        }
    }
    
    return answer;
}